blotter-template/README.md

```diff
@@ -185,7 +185,13 @@ if [ $confreading = 1 ]; then
    java -jar "$jar" "$@"
    exit $?
 else
-   java -cp "$classpath" -jar "$jar" "$@" 2> "$LOGDIR"/java-$BLADENAME.log > /dev/null &
+   if [[ ! -z $START_FOREGROUND_NOLOGS ]]; then
+       java -cp "$classpath" -jar "$jar" "$@" > "$LOGDIR"/java$BLADENAME.log 2>&1
+   elif [[ ! -z $START_FOREGROUND ]]; then
+       java -cp "$classpath" -jar "$jar" "$@" --foreground-logs
```

Comment:
I assumed that it's the same option in DS4J as DSDK but I don't know if u need the -- in front of it - can you check that out ?